cloudapi/config.go

```diff
@@ -166,6 +168,7 @@ func NewConfig() Config {
 		TracesPushInterval: types.NewNullDuration(1*time.Second, false),
 
 		MaxMetricSamplesPerPackage: null.NewInt(100000, false),
+		MaxTimeSeriesInBatch:       null.NewInt(100000, false),
```

Comment:
```suggestion
		MaxTimeSeriesInBatch:       null.NewInt(10000, false),
```
Starting with one order of magnitude lower should be better.

Reply:
Resolved with the latest force push